articles/active-directory/develop/msal-net-aad-b2c-considerations.md

```diff
@@ -68,14 +70,16 @@ Acquiring a token for an Azure AD B2C protected API in a public client applicati
 
 ```csharp
 IEnumerable<IAccount> accounts = await application.GetAccountsAsync();
-AuthenticationResult ar = await application .AcquireToken(scopes, parentWindow)
+AuthenticationResult ar = await application .AcquireTokenInteractive(scopes)
                                             .WithAccount(GetAccountByPolicy(accounts, policy))
+                                            .WithParentActivityOrWindow(ParentActivityOrWindow)
                                             .ExecuteAsync();
 ```
```

Comment:
@jennyf19 don't you think we should show the full code for the Sisu policy, which catches the password reset exception and triggers the password reset policy ?

Reply:
@jmprieur I don't think so. When. We redo our b2c docs we can consider doing that. Also, soon, I think, password reset will be under su_si user flow, so we won't need that extra code.

Reply:
thanks for confirming, @jennyf19